src/main/webapp/data/userMapData.json

```diff
@@ -1814,8 +1815,8 @@
       "St. Anne Community High School 302",
       "St. Anne Grade School",
       "St. John's Christian Academy",
-      "St. Joseph St. Thomas School",
-      "St. Joseph's Academy",
+	  "St. Joseph's Academy",
```

Comment:
The order here should still remain, since St Joseph's **A**cademy comes before St Joseph S**t** Thomas